python/catkin/init_workspace.py

```diff
@@ -87,7 +88,11 @@ def init_workspace(workspace_dir):
     # look in to-be-initialized workspace first
     src = os.path.join(workspace_dir, 'catkin', 'cmake', 'toplevel.cmake')
     if os.path.isfile(src):
-        src_file_path = os.path.relpath(src, workspace_dir)
+        if sys.platform.startswith('win32'):
```

Comment:
Other code in this repos checks for `sys.platform == 'win32'`. When does it ever contain a longer string starting with `win32`?

Reply:
according to https://docs.python.org/2/library/sys.html#sys.platform, `sys.platform` would not resolve to `win32...`; this is a style mismatch, changing to `sys.platform == 'win32'`